notebooks/time-to-merge-prediction/workshop/deployment-config.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: machinelearning.seldon.io/v1
+kind: SeldonDeployment
+metadata:
+  name: YOUR_USERNAME
+spec:
+  name: YOUR_USERNAME
+  predictors:
+  - componentSpecs:
+    - spec:
+        containers:
+        - image: quay.io/aicoe/github-pr-ttm:latest
```

Comment:
@suppathak this image was created from https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/models/openshift-origin-github-ttm/GitHubTTMModel.py, here it seems that the model returns these class labels https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/models/openshift-origin-github-ttm/GitHubTTMModel.py#L53

In our workshop each model trained by the attendee will have different class durations depending on the quantile values.
<img width="852" alt="Screen Shot 2022-07-21 at 9 54 36 AM" src="https://user-images.githubusercontent.com/32435206/180231076-622ab161-ad8f-416f-a89f-bf720d677a4f.png">
So we cannot use these labels from the openshift origin repository.

I would recommend that we create an s2i folder for the workshop where we return the class number instead of the duration like the following and create a quay image with that and link that here.
return [
            "class 0",
            "class 1",
            "class 2",
          ..........
        ]

I see [this](https://github.com/operate-first/continuous-delivery/issues/35) hasnt been provisioned yet, so we can create a repo on your personal org for this for time being.

Reply:
@oindrillac , Sure, For the time being, I will add my personal quay repo  (quay.io/supathak/github-ttm-test:v2) , where the model will return [these](https://github.com/suppathak/ttm-test/blob/master/models/github-ttm/GitHubTTMModel.py#L64) class labels.